honeybee_radiance/properties/_base.py

```diff
@@ -160,3 +163,192 @@ def ToString(self):
 
     def __repr__(self):
         return 'Base Radiance Properties:\n host: {}'.format(self.host.identifier)
+
+
+class _DynamicRadianceProperties(_RadianceProperties):
+    """Base class of Radiance Properties for all planar geometry objects.
+
+    This includes Apertures and Shades.
+
+    Args:
+        host: A honeybee_core object that hosts these properties.
+        modifier: A Honeybee Radiance Modifier for the object. If None,
+            it will be set by the parent Room ModifierSet or the Honeybee
+            default generic ModifierSet.
+        modifier_blk: A Honeybee Radiance Modifier to be used for this object
```

Comment:
Well, for static geometry, these two are the same. So that's really the case that this docstring is speaking to here since the majority of geometry in a model is usually static.

For dynamic geometry with states, you could potentially have a different `modifier_direct` for each state like you show in the radiance-folder-structure. So the `modifier_blk` property of the entire parent `Shade` or `Aperture` kinda takes a back seat to the `modifier_direct` of each state of the `Shade` or `Aperture`. The `modifier_blk` property of the entire `Aperture` is still used for isolation studies of dynamic apertures, though.